Simplenote.xcodeproj/project.pbxproj

```diff
@@ -1762,7 +1762,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "plist=\"${TARGET_BUILD_DIR}/${INFOPLIST_PATH}\"\nyear=$(date \"+%Y\")\ncopyright=\"Copyright © $year Automattic. All rights reserved.\"\n\n# Set the version numbers in the buildPlist\n/usr/libexec/PlistBuddy -c \"Set :NSHumanReadableCopyright $copyright\" $plist\n";
+			shellScript = "plist=\"${TARGET_BUILD_DIR}/${INFOPLIST_PATH}\"\nyear=$(date \"+%Y\")\ncopyright=\"Copyright © $year Automattic. All rights reserved.\"\n\n# Set the version numbers in the buildPlist\n#\n# Temporarily disabled! It's failing the build after updating to support Apple Silicon.\n#\n# Unfortunately, a number of changes were required to make the build pass before getting to this point,\n# so there is no clear responsible for this failure yet.\n# /usr/libexec/PlistBuddy -c \"Set :NSHumanReadableCopyright $copyright\" $plist\n";
```

Comment:
You'll have to scroll horizontally a bit to see the change. Here's a formatted version:

```diff
  plist="${TARGET_BUILD_DIR}/${INFOPLIST_PATH}"
  year=$(date "+%Y")
  copyright="Copyright © $year Automattic. All rights reserved."

  # Set the version numbers in the buildPlist
- # /usr/libexec/PlistBuddy -c "Set :NSHumanReadableCopyright $copyright" $plist
+ #
+ # Temporarily disabled! It's failing the build after updating to support Apple Silicon.
+ #
+ # Unfortunately, a number of changes were required to make the build pass before getting to this point,
+ # so there is no clear responsible for this failure yet.
+ # /usr/libexec/PlistBuddy -c "Set :NSHumanReadableCopyright $copyright" $plist
```

I'd be curious to see if this happens for you, too.

Reply:
This doesn't happen to me locally [or in CI](https://app.circleci.com/pipelines/github/Automattic/simplenote-macos/1830/workflows/9ab7856b-4399-4f2e-8b2a-2605425b52f4/jobs/1784), so AFAICT it's safe to put back, but LMK if not?